sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 #### Bugs Fixed
 
+* Enabled read failover to preferred locations in the case of single-write/multi-read region enabled account. See [Bug: Cosmos DB Client gets stuck in timeout retry loop](https://github.com/Azure/azure-sdk-for-java/issues/31260#issue-1396454421)
```

Comment:
the link here should be the PR link, not the issue link, please reference others

Reply:
Fixed this already